### DISCUSSION TITLE
Export

### DISCUSSION THREAD

**Discussion by EdgeTypE** [vladmandic/human#283] [Q&A] [answered]
Is there a way to export the resulting data in .fbx or similar format?

**Comment by vladmandic** [comment, marked as answer]
indirectly as having full 3d framework like that embedded would be an overkill  

you can create a `three.js` or `babylonjs` scene, draw results and then export the scene as `gltf` or `glb` with a single call  |
(or even `fbx` with a plugin)

for example on how to create a scene and draw results, see <https://github.com/vladmandic/human-motion>
